test/Microsoft.Azure.Devices.Edge.Test/PriorityQueues.cs

```diff
@@ -124,32 +128,159 @@ public async Task PriorityQueueModuleToModuleMessages()
             Assert.IsTrue(isPassed);
         }
 
-        private Dictionary<string, object> BuildRoutes(string[] priorities, string sendModule, string receiveModule)
+        [Test]
+
+        public async Task PriorityQueueTimeToLive()
+        {
+            CancellationToken token = this.TestToken;
+            string trcImage = Context.Current.TestResultCoordinatorImage.Expect(() => new ArgumentException("testResultCoordinatorImage parameter is required for Priority Queues test"));
+            string loadGenImage = Context.Current.LoadGenImage.Expect(() => new ArgumentException("loadGenImage parameter is required for Priority Queues test"));
+            string relayerImage = Context.Current.RelayerImage.Expect(() => new ArgumentException("relayerImage parameter is required for Priority Queues test"));
+
+            const string trcModuleName = "testResultCoordinator";
+            const string loadGenModuleName = "loadGenModule";
+            const string relayerModuleName = "relayerModule";
+            const string trcUrl = "http://" + trcModuleName + ":5001";
+            const string loadGenTestDuration = "00:00:20";
+
+            string routeTemplate = $"FROM /messages/modules/{loadGenModuleName}/outputs/pri{0} INTO BrokeredEndpoint('/modules/{relayerModuleName}/inputs/input1')";
+
+            string trackingId = Guid.NewGuid().ToString();
+            string priorityString = this.BuildPriorityString(5);
```

Comment:
I think you need to refactor to better manage priorityString, ttlString and routes.  It is loosely-coupled and easy to break.

Reply:
Refactored using TestInfo struct